src/object/cli_obj.c

```diff
@@ -3489,3 +3490,11 @@ dc_obj_verify(daos_handle_t oh, daos_epoch_t *epochs, unsigned int nr)
 
 	return rc;
 }
+
+int
+dc_obj_anchor_split(uint32_t shard, daos_anchor_t *anchor) {
+	daos_anchor_set_zero(anchor);
+	dc_obj_shard2anchor(anchor, shard);
+	daos_anchor_set_flags(anchor, DIOF_TO_SPEC_SHARD);
```

Comment:
Does it return DAOS_ANCHOR_TYPE_EOF when we reach the end of the shard?

Reply:
yes, but it required that fix i made above on line 2156